recipes-support/rcheevos/rcheevos.bb

```diff
@@ -0,0 +1,22 @@
+SUMMARY = "rcheevos is a set of C code that tries to make it easier for emulators to process RetroAchievements data."
+HOMEPAGE = "https://github.com/RetroAchievements/rcheevos"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=466b936a0a89a7677882b9a92c189c2f"
+
+
+SRCREV = "203fc282d542a058241f3c5a28a2d0ca15e628cb"
+PV = "9.2.0"
+S = "${WORKDIR}/git"
+
+SRC_URI = " \
+	git://github.com/RetroAchievements/rcheevos.git;nobranch=1;protocol=https \
+	file://CMakeLists.txt \	
```

Comment:
Maybe this should be moved to upstream?

Reply:
That would be nice, and I wanted to do that. But the situation with Kodi rcheevos support is that Kodi uses a version of rcheevos that is about two years old. I also created a CMakeLists.txt file for rcheevos v10.4.0 and it is quite different from what it needs for v9.2.0, the version Kodi wants. 
Rcheevos Upstream no longer has support for the 9.x versions,  so there is no place to push it.

Reply:
ok, I think we may progress with it as is.

Reply:
great!

I'll see what it takes to build kodi-addon-game-libretro with rcheevos 10. But if I succeed, I'll send that as a follow-up, since it will probably need some testing before.